tests/integration/testdata/traces/python-apigw-sfn/template.yaml

```diff
@@ -0,0 +1,118 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
I wonder if we can use same projects for both `sam logs` and `sam traces` integration tests